testing/trino-testing/src/main/java/io/trino/testing/BaseFailureRecoveryTest.java

```diff
@@ -812,14 +812,13 @@ private static StageStats getRootStage(MaterializedResult result)
         return requireNonNull(statementStats.getRootStage(), "root stage is null");
     }
 
-    private Session enableDynamicFiltering(boolean enabled)
+    protected Session enableDynamicFiltering(boolean enabled)
     {
         Session defaultSession = getQueryRunner().getDefaultSession();
         return Session.builder(defaultSession)
                 .setSystemProperty(ENABLE_DYNAMIC_FILTERING, Boolean.toString(enabled))
                 .setSystemProperty(JOIN_REORDERING_STRATEGY, NONE.name())
                 .setSystemProperty(JOIN_DISTRIBUTION_TYPE, PARTITIONED.name())
-                .setCatalogSessionProperty(defaultSession.getCatalog().orElseThrow(), "dynamic_filtering_wait_timeout", "1h")
```

Comment:
This was added in https://github.com/trinodb/trino/commit/504c9cfcfd863fb7a76ecef0a748a51ff1992579 ....